### PR TITLE
Adding force remove instructions for powered-off node (BSC#1173235)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -207,6 +207,18 @@ placement:
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/minions remove ses-min2</screen>
    </step>
   </procedure>
+  <important>
+    <para>
+      In the event of a failure and the minion you are trying to remove is
+      in permanently powered-off, you will need to remove the node from the
+      &smaster;:
+    </para>
+<screen>&prompt.cephuser;salt-key -d <replaceable>minion_id</replaceable></screen>
```
Comment (ricardoasmarques) [inline xml/admin_saltcluster.xml:216]: should be `&prompt.smaster;`

**Review by ricardoasmarques — APPROVED** [review]
lgtm

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -207,6 +207,18 @@ placement:
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/minions remove ses-min2</screen>
    </step>
   </procedure>
+  <important>
+    <para>
+      In the event of a failure and the minion you are trying to remove is
+      in permanently powered-off, you will need to remove the node from the
```
Comment (lproven) [inline xml/admin_saltcluster.xml:213]: > is in permanently powered-off

...?

I am not sure what is meant here. Either:
"is in a permanently-powered-off state"
or
"is permanently powered-off."

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/admin_saltcluster.xml (hunk 2 above)
Comment (asettle) [reply xml/admin_saltcluster.xml:213]: Looks like I changed my mind halfway. Option A, sir.

**Review by lproven — APPROVED** [review]

**Pull request merged**